lib/sidekiq_flow/task.rb

```diff
@@ -9,7 +9,7 @@ class Task < Model
 
     def self.attribute_names
       [
-        :start_date, :end_date, :loop_interval, :retries, :queue,
+        :start_date, :end_date, :loop_interval, :retries, :queue, :inline,
```

Comment:
🤔 maybe with can have a `performed` attribute also we set to the parent task so that it's a bit safer than to assume idempotency on the task code. And if the child raises and the parent task is retried, we just skip the parent if `performed` is true.

Reply:
It already exists `status: succeeded` :-)
I'll check that